docs/arch/adr-29.md

```diff
@@ -0,0 +1,31 @@
+# ADR 29: Video Player
+
+Created: Dec 8, 2020
+
+## Context
+
+Future projects using the `front-end-monorepo` will require a video player. Some projects may simply need a video to play. Some projects will require users to interact with a video file; such as make annotations on the video to mark locations or item sizes. The video player must be highly customizable, able to be used in a React code base project and support all major browsers.
+
+## Decision
+
+We will implement `react-player` [Github](https://github.com/CookPete/react-player).
+Demo Page: [Demo](https://cookpete.com/react-player/)
```

Comment:
Could we have a bit more information here about how `react-player` will enable us to customize the video player as needed for project requirements versus using the native HTML video player and browser APIs?